src/engine/enginebuffer.cpp

```diff
@@ -493,7 +493,7 @@ void EngineBuffer::slotTrackLoading() {
 }
 
 TrackPointer EngineBuffer::loadFakeTrack(double filebpm) {
-    TrackPointer pTrack(new TrackInfoObject(), &QObject::deleteLater);
+    TrackPointer pTrack(TrackInfoObject::newTemporary());
```

Comment:
I think ther is a naming missmatch FakeTrack vs Temporary. 
Is this temorary only used for testing? Then "Dummy", which is was in one of your earlyer commits will work well for me.